### PR TITLE
January News Update

### PR DESCRIPTION
**Pull request by Cerebulon**
Adds IC news articles to the newscaster:
1. Sif Governor Bjorn Arielsson to Retire
2. Bjorn Arielsson Issues "Farewell Address"
3. Farewell Address Transcript
4. Moravec Nephew Announces Vir Governor Candidacy
5. Vani Jee Orbital Tour Cut Short

### CODE REVIEW

File: code/modules/lore_codex/news_data/main.dm [hunk 1]
```diff
@@ -339,4 +344,60 @@
...
+	<br><br>\
+	Others have criticized the speech's seemingly communist tone, with Arielsson expressing approval for the socialist Free Relan Federation and the anarchist Casini's Reach despite opposing their secessionist ideals. Still others claim that the speech offered 'nothing but empty feelings' and that it lacked specific, actionable resolutions on the growing secessionist movement in VirGov. Some have even framed the Address as a form of political maneuvering by the venerable politician, claiming that he voiced unpopular sentiments specifically to hamper the Shadow Coalition's re-election bid after well-publicized disagreements with SC party bosses.\
+	<br><br>\
+	The actions of Arielsson and Vir's proximity to the border have lead to increased focus on the upcoming Gubernatorial  election on a nationwide level, with the Icarus Front alone projected to spend upwards of a billion thalers on publicity. Minor party candidates like the former MLM member Luisa Hannirsdottir and the Mercurial Phaedrus already have strong support in the polls, promising a fierce election that could ultimately tip power in the system in any direction.\
```
Comment (Atermonera) [inline code/modules/lore_codex/news_data/main.dm]: Double space after `Gubernatorial`

File: code/modules/lore_codex/news_data/main.dm [hunk 2]
```diff
@@ -339,4 +344,60 @@
...
+	<br><br>\
+	A full excerpt is available below."
+	children = list(
+		/datum/lore/codex/page/bjornretirement
+		)
+
+/datum/lore/codex/page/bjornretirement
+	name = "Bjorn Aielsson Farewall Address"
```
Comment (Atermonera) [inline code/modules/lore_codex/news_data/main.dm]: `Aielsson`
`Farewall`

File: code/modules/lore_codex/news_data/main.dm [hunk 3]
```diff
@@ -339,4 +344,60 @@
...
+
+/datum/lore/codex/page/article32
+	name = "01/25/63 - Moravec Nephew Announces Vir Governor Candidacy"
+	data = "The Sol Economic Organization has announced that Calvert Moravec, nephew of NanoTrasen CEO Albary Moravec will be running under their ticket in the upcoming Vir gubernatorial elections. Calvert has stated that he will run on a pro-business platform, and has chosen Vir to do so due to the 'Unique beauty and economic prospects of an interstellar crossroads such as Vir'.\
+	<br><br>\
+	Despite being a lifelong resident of Alpha Centauri, Moravec was recently approved for Sif citizenship, making him eligible for local candidacy and has reportedly moved into a luxurious New Reykjavik penthouse. Perhaps best known for his soaring stock market investment success over the previous few years, Calvert's first foray into politics is not wholly unexpected as the Moravec family has long leveraged their wealth in international affairs, though successful election would mark their first sitting member of the Colonial Assembly. Fellow SEO candidate Councillor Hal Wekstrom has expressed his full support for Moravec.\
```
Comment (Atermonera) [inline code/modules/lore_codex/news_data/main.dm]: Sif citizenship or Vir citizenship?

File: code/modules/lore_codex/news_data/main.dm [hunk 4]
```diff
@@ -339,4 +344,60 @@
...
+	<br><br>\
+	Three candidates will be elected as representatives to the Colonial Assembly later this year, with the most popular also attaining the position of system governor."
+
+/datum/lore/codex/page/article33
+	name = "01/27/63 - Vani Jee Orbital Tour Cut Short"
+	data = "Icarus Front Representative Candidate Vani Jee has delayed the remainder of her campaign tour of orbital colonies and outposts around the Vir system after an alleged altercation with NanoTrasen security.\
+	Candidate Jee had been visiting the NLS Southern Cross, a NanoTrasen station in Sif orbit to recieve a corporate tour and meet with voters, when her concluding question and answer session was interrupted by hecklers, leading to the event being cut short. Jee alleges that footage of the event was seized by NanoTrasen corporate security and has accused the trans-stellar corporation of the intentional intimidation of Icarus Front and Sol Economic Organization candidates in what she describes as 'a clear display of corruption in favour of company-favourite Calvert Moravec', though she has praised the individual employees of the Cross for their hospitality and thought-provoking questions.\
```
Comment (Atermonera) [inline code/modules/lore_codex/news_data/main.dm]: `receive` a corporate tour

Comment (Atermonera) [reply code/modules/lore_codex/news_data/main.dm]: Also wouldn't they be favoring SEO candidates over Shadow Coalition canditions, given that SEO is basically TSC's: the party?

Comment (Cerebulon) [reply code/modules/lore_codex/news_data/main.dm]: Ah you're right that should be SC not SEO (It's specifically referring to the candidate who is literally related to the NT CEO)

**Pull request merged**